### ISSUE TITLE
Add a "Time Requested" Column

### ISSUE THREAD

**Issue by surajpkhetani** [nccgroup/AutoRepeater#58]
It will be great to have the "Time Requested" column to better analyze the requests.

**Comment by Nexsus1985** [comment]
I agree this is important feature. It will also allow testing for all time based attacks.

Thanks.
